package.json

```diff
@@ -2,13 +2,32 @@
   "name": "@solana-developers/helpers",
   "version": "2.5.2",
   "description": "Solana helper functions",
-  "main": "dist/index.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/esm/index.d.ts",
   "type": "module",
   "private": false,
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./node": {
+      "import": "./dist/esm/node.js",
+      "require": "./dist/cjs/node.js"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "sideEffects": false,
```

Comment:
What is this? It's not documented in https://docs.npmjs.com/cli/v10/configuring-npm/package-json

Reply:
From Ayush: https://github.com/webpack/webpack/blob/main/examples/side-effects/README.md